profiles/api_test.py

```diff
@@ -53,17 +57,31 @@ def test_one_social(self):
 
     def test_two_social(self):
         """
-        get_social_username should return latest social user name if
-        there are two social edX accounts for a user
+        get_social_username should return None if there are two social edX accounts for a user
         """
-        new_social_auth = UserSocialAuthFactory.create(user=self.user, uid='other name')
-        assert get_social_username(self.user) == new_social_auth.uid
+        self.user.social_auth.create(
+            provider=EdxOrgOAuth2.name,
+            uid='other name',
+        )
```

Comment:
`UserSocialAuthFactory` can/should still be used here (`UserSocialAuthFactory.create(user=self.user, uid='other name')`)